src/main/java/Duke.java

```diff
@@ -1,10 +1,44 @@
+import java.util.Optional;
+import java.util.Scanner;
+
 public class Duke {
+    private static String entries = "../data/entries.txt";
+
     public static void main(String[] args) {
-        String logo = " ____        _        \n"
-                + "|  _ \\ _   _| | _____ \n"
-                + "| | | | | | | |/ / _ \\\n"
-                + "| |_| | |_| |   <  __/\n"
-                + "|____/ \\__,_|_|\\_\\___|\n";
-        System.out.println("Hello from\n" + logo);
+        // set up
+        TaskList taskList = TaskList.instanceOf();
+        Optional<Storage> desiredStorage = Optional.empty();
+        try {
+            desiredStorage = Optional.of(new Storage(entries));
+        } catch (Exception e) {
+            Ui.showError(e);
+        }
+        Optional<Storage> storage = desiredStorage;
+        storage.ifPresent(s -> taskList.addPreviousTasks(s.load()));
+
+        // greet user
+        Ui.sayHi();
```

Comment:
comment can be made redundant if you renamed sayHi to greetUser.

Reply:
Haha. I just forgot to delete the comment 😅 Thanks though. I'll remove it later. (: